docs/nuget-org/organizations-on-nuget-org.md

```diff
@@ -71,11 +71,11 @@ If you wish to transfer some of your packages to a newly created organization, y
 
 ## Publishing packages
 
-You publish packages to an organization like you publish packages to a user account: by directly uploading the package to nuget.org or by pushing the package through the `nuget push` or `dotnet nuget push` CLI commands.
+You publish packages to an organization like you publish packages to a user account: by directly uploading the package to NuGet.org or by pushing the package through the `nuget push` or `dotnet nuget push` CLI commands.
 
 ### Uploading packages
 
-When you directly upload a new package on the [nuget.org Upload](https://www.nuget.org/packages/manage/upload) page, you assign the package owner to a user or organization account :
+When you directly upload a new package on the [NuGet.org Upload](https://www.nuget.org/packages/manage/upload) page, you assign the package owner to a user or organization account :
 
 ![Upload package with account option](media/org-upload-option.png)
 
```

Comment:
because I can't comment on that line
### Deleting an organization account
@Mikejo5000 this is now possible. Can you please go through the workflow and add the screenshots as well.

Reply:
Added a short section